### ISSUE TITLE
New wizard does not allow sufficient space for short item descriptions

### ISSUE THREAD

**Issue by mrlacey** [Microsoft/WindowsTemplateStudio#1849]
As per images below

![image](https://user-images.githubusercontent.com/189547/35934589-ec09d1ee-0c35-11e8-9020-3255438f8c94.png)

![image](https://user-images.githubusercontent.com/189547/35934612-f7e83a82-0c35-11e8-88da-64cc0fb47ba4.png)

![image](https://user-images.githubusercontent.com/189547/35934654-0c90d944-0c36-11e8-8b8a-5aaec991a58f.png)

![image](https://user-images.githubusercontent.com/189547/35934684-23f4de8c-0c36-11e8-8ddc-6d8fea1e4b60.png)

Either need to increase the amount of space available, make text smaller, or rewrite the text to be shorter.

Making it smaller will impact legibility.
Rewriting to make it fit will need to allow for all translated versions to also be written with the amount of display space in mind.
Unless there's another option this means it will be necessary to increase the amount of available space to make the text fully visible.

**Comment by ralarcon** [comment]
Regarding the Frameworks and Project Types, we need to ensure we are using the "short" description. For the other, we can try to reduce the font size by 1px or review margins etc.

**Comment by mvegaca** [comment]
We were using long descriptions in card layout, I've fixed in #1851 
I think this should be the final solution, 4 lines are enough text to show on the card, If someone wants to read more can click on details button.

**Comment by sibille** [comment]
Study with design team if we could show more text on card items, if not adjust the texts to make them fit.

**Comment by mvegaca** [comment]
Check fixed on
Templates version: 0.16.18047.4
Wizard version: 0.16.18047.1
